src/pybind/mgr/dashboard_v2/controllers/pool.py

```diff
@@ -0,0 +1,22 @@
+from functools import partial
+
+from ..models.nodb import nodb_context
+from ..models.pool import CephPool
+from ..tools import ApiController, RESTController
+
+
+def nodb_serializer(model, obj):
```

Comment:
Can you move `nodb_serializer` to `..models.nodb` and just import it here and in `.cluster`?

Reply:
Yes, sure